client/state/user-devices/reducer.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	USER_DEVICES_ADD,
+} from 'state/action-types';
+
+export default createReducer( {}, {
+	[ USER_DEVICES_ADD ]: ( state, { devices } ) => devices
```

Comment:
@gwwar I'm having mixed feelings about this. Would it be better to do something like this instead?

```js
export const devices = ( state = null, { devices } ) =>
  USER_DEVICES_ADD === type
    ? devices
    : state;

export default devices;
```